api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1257,4 +1257,18 @@ message TrackClusterStats {
   // <config_cluster_manager_cluster_stats_request_response_sizes>`  tracking header and body sizes
   // of requests and responses will be published.
   bool request_response_sizes = 2;
+
+  // If true, some stats will be emitted per-endpoint, similar to the stats in admin ``/clusters``
+  // output.
+  //
+  // This does not currently output correct stats during a hot-restart.
+  //
+  // This is not currently implemented by all stat sinks.
+  //
+  // These stats do not honor filtering or tag extraction rules in :ref:`StatsConfig
```

Comment:
Would there be filtering in sinks anyway?

It looks from the impl that these *will* have the configured tags, is that right? Just not the extracted tags?

Reply:
update: it looks to me cluster stats *will* be emitted in the /stats endpoint, and will/could/should be filtered using the existing regex. WDYT?

Reply:
This needs to be clarified. What doesn't work is the bootstrap-config for filtering out stats (so that they don't exist at all). regex-filtering on admin still works, and fixed-tags work (but not extracted tags). I'll clarify this.